### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by markedwardmurray**
@jmburges @misterfifths 

regarding https://github.com/learn-co-curriculum/mobile-development-with-ios-track/issues/15

please review by Wed 9/16/2015

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,263 @@
...
+128
+Fizz
+Buzz
+```
+
+## Advanced
+
+1. The FizzBuzz exercise is primarily about best practice and efficiency. Can you identify some of the problems with the implementation of FizzBuzz that we wrote? For starters, it repeats itself, both in the conditional checks, and in the instructions. Can you come up with a better logical structure that doesn't repeat itself?
+
+2. Furthermore, while we did document the starting and ending points for the loop by saving the values into the `start` and `limit` variables, the remainder (`%`) calculations are still written with "magic numbers". Document these values by creating new variables called `fizz` and `buzz` that are set to `3` and `5` respectively.
```
Comment (misterfifths) [inline README.md]: Hmm I don't like `fizz` and `buzz` as names. Maybe `fizzDivisor` and `buzzDivisor`?

File: objc-fizzbuzz/FISAppDelegate.m [hunk 2]
```diff
@@ -1,45 +1,85 @@
...
-    // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    
+    /*
+     
+     * Write your code here!
+     
+     */
+    
+    // Basic
+    NSUInteger start = 100;
```
Comment (misterfifths) [inline objc-fizzbuzz/FISAppDelegate.m:17]: I don't know what your process is looking like these days, but is this bound for the solution branch?

Comment (markedwardmurray) [inline objc-fizzbuzz/FISAppDelegate.m:17]: looks like it, yeah, as intended. i usually write the solution branch first, then have a workflow to bring everything except the solution elements into master.

File: README.md (hunk 1 above)
Comment (markedwardmurray) [inline README.md]: `fizzIncrement` and `buzzIncrement` ?

Comment (misterfifths) [inline README.md]: Sure

**Pull request merged**